advocacy_docs/partner_docs/EsriArcGISProandArcGISEnterprise/04-ConfiguringEsriArcGISProandEsriArcGISEnterprise.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Configuring Esri ArcGIS Pro and Esri ArcGIS Enterprise'
+description: 'Walkthrough on configuring Esri ArcGIS Pro and Esri ArcGIS Enterprise'
+---
+
+Implementing Esri ArcGIS Pro with EDB Postgres Advanced Server requires the following components:
+
+- A running EDB Postgres Advanced Server instance
+- A current version of PostGIS installed on EDB Postgres Advanced Server
```

Comment:
saying a "A current version" is to vague.   We could say 3.2 or above

Reply:
Rephrased to read- "PostGIS version 3.2 or above, installed on EDB Postgres Advanced Server"